docs/src/content/pages/fields/array.mdoc

```diff
@@ -51,6 +51,34 @@ complexArray: fields.array(
 ),
 ```
 
+## Slug field
+
+The array field's `slugField` option is useful to replace indexes normally in file paths for images or documents e.g.
```

Comment:
Is there an example missing at the end?

Reply:
Nah it was just written badly, I've now changed to `etc` at the end. But unsure if this section makes sense so feel free to suggest improvements if you can think of any 🙌🏻